src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
```

Comment:
I think there is no need to add a parameter as health because it is a default property.

Reply:
optional